.github/workflows/test.yml

```diff
@@ -217,7 +217,7 @@ jobs:
         MATRIX_CONTEXT: ${{ toJson(matrix) }}
       with:
         status: ${{ job.status }}
-        channel: "#terra-wsm-alerts"
+        channel: "#dsp-workspace-test-alerts"
```

Comment:
Notify on failure, non-bumper push. Also notifies in previous step to #dsde-qa always if alpha/staging.